Fix build failure on alpha:
  plt.c:32:12: error: dereferencing pointer to incomplete type ‘struct library_symbol’
    return sym->enter_addr;
            ^
diff -U0 -raN ltrace-0.7.3/sysdeps/linux-gnu/alpha/plt.c ltrace-0.7.3/sysdeps/linux-gnu/alpha/plt.c
--- ltrace-0.7.3_p4/work/ltrace-0.7.3/sysdeps/linux-gnu/alpha/plt.c     2012-11-10 13:43:55.000000000 +0100
+++ foo/work/ltrace-0.7.3/sysdeps/linux-gnu/alpha/plt.c 2016-06-27 23:21:28.000000000 +0200
@@ -23,0 +24 @@
+#include "library.h"
diff -U0 -raN ltrace-0.7.3/sysdeps/linux-gnu/alpha/trace.c ltrace-0.7.3/sysdeps/linux-gnu/alpha/trace.c
--- ltrace-0.7.3_p4/work/ltrace-0.7.3/sysdeps/linux-gnu/alpha/trace.c   2016-06-27 23:30:08.000000000 +0200
+++ foo/work/ltrace-0.7.3/sysdeps/linux-gnu/alpha/trace.c       2016-06-27 23:25:20.000000000 +0200
@@ -29,0 +30 @@
+#include "backend.h"
